src/wdio/services/ServeStaticService.js

```diff
@@ -2,19 +2,74 @@ import serveStatic from '../../../scripts/serve/serve-static';
 import SERVICE_DEFAULTS from '../../../config/wdio/services.default-config';
 import Logger from '../../../scripts/utils/logger';
 
+const WebpackDevServer = require('webpack-dev-server');
+const webpack = require('webpack');
+const ip = require('ip');
+
 const { serveStatic: SERVE_STATIC_DEFAULTS } = SERVICE_DEFAULTS;
 const context = '[Terra-Toolkit:serve-static-service]';
 
+const displayServer = (localAddress, networkAddress) => {
+  Logger.log('Server started listening at', { context });
+  Logger.log(`* Local:            ${Logger.emphasis(localAddress)}`);
+  Logger.log(`* On your network:  ${Logger.emphasis(networkAddress)}`);
+};
+
+// Create a webpack dev server instance.
+const wds = (options) => {
+  const {
+    port, index, locale,
+  } = options;
+  const host = '0.0.0.0';
+  let { config } = options;
+  // if config is a function, execute it with prod mode if applicable.
+  if (typeof config === 'function') {
+    const env = { ...locale && { defaultLocale: locale } };
+    config = config(env, { p: true });
+  }
+
+  // pull the dev server options out of the webpack config. override host, port, and stats. SRY.
+  const devServerOptions = Object.assign({}, config.devServer, {
+    // Disable hot reloading
+    hot: false,
+    inline: false,
+    host,
+    port,
+    index,
+    stats: {
```

Comment:
```
    host,
    port,
    index,
```
Are these values we should be defining in webpack or are we using the defaults provided by webpack-dev-server?

Reply:
specifically these are defined by the wdio config